metriks/gauge.go

```diff
@@ -0,0 +1,113 @@
+package metriks
+
+import (
+	"context"
+	"sync/atomic"
+	"time"
+
+	"github.com/armon/go-metrics"
+	"github.com/netlify/netlify-commons/util"
+)
+
+const (
+	defaultGaugeDuration = time.Second * 10
```

Comment:
So if I look at our custom metrics, it looks like most of them report on a 10sec frequency. So best practice sampling says we should measure at N/2 or 5 secs to avoid the case where we don't report in a given interval. I realize this is configurable, but maybe best to start with a default that would work in our case?

Reply:
good call ~ updated!